app.js

```diff
@@ -162,3 +162,9 @@ TeamValidator.PM.spawn();
  *********************************************************/
 
 require('./repl').start('app', cmd => eval(cmd));
+
+/*********************************************************
+ * Load the GitHub plugin
+ *********************************************************/
+
+Object.assign(Chat.commands, require('./chat-plugins/github').commands);
```

Comment:
Isn't it still called during a hotpatch, though? How does this code prevent it from being called during a hotpatch?